### ISSUE TITLE
Default Callsign/name

### ISSUE THREAD

**Issue by jduff001** [Othernet-Project/dreamcatcher-chat#3]
Could the default be changed from "Anonymous" to a unique name, possibly
a partial of the mac address... this might allow the same SD card image to be
placed in multiple dreamcatchers, then avoid any confusion if the user fails to 
change the Callsign at the ipxxx.xxx.xxx.xxx:8080 initial parameters screen.

**Comment by foxbunny** [comment]
So basically not a completely random name, but something tied to the device itself, is what you are proposing?

**Comment by jduff001** [comment]
Yes,  ideally,  the user would update it correctly to their callsign,  but if they don't the conversations look odd with Anonymous chatting with other Anonymous users.

**Comment by foxbunny** [comment]
Yeah, I got a little lazy with that one, didn't I. :D

**Comment by jduff001** [comment]
I guess saving all the settings in the configuration page would be nice, so the user wouldn't have verify frequency / bw / sf / cr each time the chat screen is started.  maybe done when the connect or disconnect boxes are checked.

**Comment by foxbunny** [comment]
Would you mind opening a new issue for that?
